### PR TITLE
Update geo widgets documentation to reflect Collect v1.23

### LINKED ISSUE #968: Update "Location widgets" section
After the UI changes to the geo widgets are complete, the screenshots and some of the descriptions in [the "Location widgets" section](https://docs.opendatakit.org/form-question-types/#location-widgets) will need to be updated to match.

### PR DESCRIPTION
**Pull request by lognaturel**
closes #968

I optimized all images.

I ran it and verified that images show up and that links work as expected.

I incorporated all of @zestyping's suggestions. Note that there is a lot of redundancy between the geotrace and geoshape widgets. Once this is merged, I'll file an issue to design a geo section that extracts a lot of this commonality.

<details>
<summary>WIP description</summary>
Currently this updates the two geopoint variants with maps and geotrace. Geoshape will look nearly identical to geotrace so I wanted to get feedback before continuing. At some point I think we should consider pulling these out to a geo section and documenting commonalities once but I'd rather not do that now because it will take more time and discussion. For now, the focus is to make sure the documentation as it exists matches reality.

One thing I did was remove most of the screenshots. [The currently-published docs](https://docs.opendatakit.org/form-question-types/#geotrace-widget) have a lot of them and I don't think they're helpful in explaining the usage and behavior of the widgets. I'd like feedback on that before I do the same for geoshape.
</details>

CC @zestyping @yanokwa

### CODE REVIEW

**Comment by zestyping** [comment]
Also, as you mentioned, some of these questions might require more time and discussion than we want to take right now and it's okay if we want to postpone those discussions to the future.

Regarding screenshots—I agree there are way too many screenshots right now and most of them don't help.  If having (and maintaining) screenshots were effortless, I might want to keep a couple of them (one showing the buttons and one showing the placement-mode dialog).  But since they do get out of date over time, I'm also fine with not having any screenshots at all.

File: odk1-src/form-question-types.rst [hunk 1]
```diff
@@ -1275,10 +1277,14 @@ appearance
 
 The default :ref:`geopoint-widget` does not allow the user to select a location.
 
-A geopoint with the appearance attribute :tc:`placement-map` allows the user to select a geopoint from a map.
+A geopoint with the appearance attribute :tc:`placement-map` allows the user to select a geopoint from a map. The user can either long press to select any location or, if the current location is known, tap on the "add marker" button at the top right of the screen. The selected location is represented by a small circle with a red outline (see arrow in screenshot).
+
+The save button saves the current location and returns to the question screen. If the location was selected by long pressing, the accuracy and altitude will both be 0. If the user's current location was selected, the accuracy will be greater than 0.
```
[the remark this answers is not in the document]
Comment (lognaturel) [reply odk1-src/form-question-types.rst]: Oh dear, thanks for this, it made me realize there is zero mention of the geopoint format in these user-facing docs.

**Comment by lognaturel** [comment]
Thanks very much, @zestyping. Strongly agree with your terminology suggestion and I tried to capture it as best as I could.

I realized there was no documentation of the "geopoint format" in these docs. I put it in the "Location widget" intro for now realizing that it's not the most accessible but I think we can rework that in another pass.

File: odk1-src/form-question-types.rst [hunk 2]
```diff
@@ -1275,16 +1287,16 @@ type
 appearance
   :tc:`placement-map`
 
-The default :ref:`geopoint-widget` does not allow the user to select a location.
+The default :ref:`geopoint-widget` does not allow the user to select a location other than the device's current geolocation.
```
Comment (zestyping) [inline odk1-src/form-question-types.rst]: select a location other than -> place the point anywhere other than

File: odk1-src/form-question-types.rst [hunk 3]
```diff
@@ -1275,16 +1287,16 @@ type
...
 
-A geopoint with the appearance attribute :tc:`placement-map` allows the user to select a geopoint from a map. The user can either long press to select any location or, if the current location is known, tap on the "add marker" button at the top right of the screen. The selected location is represented by a small circle with a red outline (see arrow in screenshot).
+A geopoint with the appearance attribute :tc:`placement-map` allows the user to select any point from a map. The user can either long press to select any location or, if the device knows its geolocation, tap on the "add point" button at the top right of the screen. The selected point is represented by a small circle with a red outline (see arrow in screenshot).
```
Comment (zestyping) [inline odk1-src/form-question-types.rst]: select any location or, if... -> place the point anywhere, or, if...

File: odk1-src/form-question-types.rst (hunk 2 above)
Comment (lognaturel) [reply odk1-src/form-question-types.rst]: Eagle-eyed @zestyping!

**Comment by lognaturel** [comment]
Thanks, @zestyping! Next up I need to fix up geoshape so I'm leaving the WIP label for now.

**Pull request merged**